content/influxdb/v2.0/visualize-data/visualization-types/band.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Band visualization
+list_title: Band
+list_image: /img/influxdb/2-0-visualizations-Band-example.png
+description: >
+  The Band view displays the single value most recent value for a time series in a Band view.
+weight: 206
+menu:
+  influxdb_2_0:
+    name: band
+    parent: Visualization types
+---
+
+The **Band** visualization displays the upper and lower boundaries for groups of data over time.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-Band-example.png" alt="Band example" />}}
+
+Select the **Band** option from the visualization dropdown in the upper left.
```

Comment:
To focus on the task, I'd prob start with "To view a band... (I notice this exists across the board, so prob a global update for later)